Chrome/background.js

```diff
@@ -145,14 +145,14 @@ chrome.runtime.onMessage.addListener(
 				if (request.openOrder === 'commentsfirst') {
 					// only open a second tab if the link is different...
 					if (request.linkURL !== request.commentsURL) {
-						chrome.tabs.create({url: request.commentsURL, selected: button, index: newIndex, openerTabId: sender.tab.id});
+						chrome.tabs.create({url: request.commentsURL, selected: button, index: newIndex++, openerTabId: sender.tab.id});
```

Comment:
Should this be `newIndex++` or `++newIndex`? In any case, I think it'd be much clearer if we moved the increment right before or after the line.